vllm/model_executor/layers/quantization/compressed_tensors/utils.py

```diff
@@ -76,25 +77,106 @@ class QuantizationArgs(BaseModel):
     )
 
 
-def find_first_name_or_class_match(
-        name: str,
-        module: Module,
-        targets: Iterable[str],
-        check_contains: bool = False) -> Optional[str]:
+# fused_name: List[shard_name]
+_FUSED_LAYER_NAME_MAPPING = {
+    "qkv_proj": ["q_proj", "k_proj", "v_proj"],
+    "gate_up_proj": ["gate_proj", "up_proj"]
```

Comment:
nit: Since this is based on model definitions, it would be nice to register and query this from the model architecture class.

Reply:
I will do this in the PR that applies the naming through all the models